Hmwrk_5(list_and_tuple)/task_1.py

```diff
@@ -0,0 +1,11 @@
+text = 'Some men interpret nine memos'
+text_1 = text.lower()
+text_2 = text_1.replace(' ', '')
+list_1 = list()
+list_1.extend(text_2)
+list_reverse = list(reversed(list_1))
+
+if list_1 == list_reverse:
+    print('It\'s a pallindrome')
+else:
+    print('It\'s not a pallindrome')
```

Comment:
A lot of unnecessary code
`actual_str = input("Enter the string: ")
print(f'Is string {actual_str} palindrome? {actual_str == actual_str[::-1]}')`